From: Gerald Schaefer <geraldsc@de.ibm.com>
Subject: IBM Codestream linux-2.6.15 october2005, patch 02-37

  Description: kernel: make cmm related proc entries world readable.
  Symptom:     Permission denied when trying to read from /proc/sys/vm/cmm_*
  Problem:     The permission bits for the cmm entries in the /proc filesystem
               are set to 0600.
  Solution:    Set the permission bits to 0644.
  Problem-ID:  22497
  Archs:       s390-31, s390-64

Acked-by: Hannes Reinecke <hare@suse.de>

--- linux-2.5/arch/s390/mm/cmm.c	12 May 2005 13:17:34 -0000	1.17
+++ linux-2.5/arch/s390/mm/cmm.c	17 Mar 2006 12:47:15 -0000	1.17.6.1
@@ -339,19 +339,19 @@
 	{
 		.ctl_name	= VM_CMM_PAGES,
 		.procname	= "cmm_pages",
-		.mode		= 0600,
+		.mode		= 0644,
 		.proc_handler	= &cmm_pages_handler,
 	},
 	{
 		.ctl_name	= VM_CMM_TIMED_PAGES,
 		.procname	= "cmm_timed_pages",
-		.mode		= 0600,
+		.mode		= 0644,
 		.proc_handler	= &cmm_pages_handler,
 	},
 	{
 		.ctl_name	= VM_CMM_TIMEOUT,
 		.procname	= "cmm_timeout",
-		.mode		= 0600,
+		.mode		= 0644,
 		.proc_handler	= &cmm_timeout_handler,
 	},
 	{ .ctl_name = 0 }

